sno/working_copy/postgis.py

```diff
@@ -251,11 +284,17 @@ def create(self):
                 SECURITY DEFINER
             """
                 ).format(
-                    func=Identifier(self.sno_schema, "track_trigger"),
+                    func=Identifier(self.schema, "_sno_track_trigger"),
                     tracking_table=self.TRACKING_TABLE,
                 )
             )
 
+    def delete(self):
+        """ Delete all tables in the schema"""
+        with self.session() as db:
+            dbcur = db.cursor()
+            dbcur.execute(SQL("DROP SCHEMA {} CASCADE").format(Identifier(self.schema)))
```

Comment:
mmmmm... not sure we want to cascade drops by default.

if someone makes a foreign key to a sno dataset table, and then we attempt to drop it for whatever reason, we shouldn't erase data from their table too. It might get the repo semi-stuck, but that's better than clearing user data.

would prefer to drop tables individually (with `RESTRICT` to prevent any issues with related tables), then drop the schema afterwards.

Reply:
That is a good plan - not required for a demo however, so I'll add a TODO.